### ISSUE TITLE
Infinite scroll in chats with media messages is a bit shitty

### ISSUE THREAD

**Issue by aesedepece** [loqui/im#250]
We have yet to figure out a better infinite scroll algorithm. There must be thousands of snippets and libraries for this, so let's do some research.

**Comment by Gioyik** [comment]
@aesedepece en la v0.3.0 no es que hay algo que muestra solo 25 mensajes y en la parte de arriba dice "mostrar mas"? Es asi o lo he imaginado?

**Comment by aesedepece** [comment]
Sí, corta cada 30 mensajes que es como están almacenados en los llamados "chunks". La longitud se puede regular en `App.defaults.Chat.chunksize`.

Ahora mismo una buena opción sería que en cuanto ese botón entre en el área visible empieze a cargarse el anterior chunk y en cuanto esté listo lo pinte.
